src/main/java/org/opentripplanner/routing/alertpatch/TransitAlert.java

```diff
@@ -15,6 +15,14 @@
 import org.opentripplanner.transit.model.framework.FeedScopedId;
 import org.opentripplanner.transit.model.framework.TransitBuilder;
 
+/**
+ * Internal representation of a GTFS-RT Service Alert or SIRI Situation Exchange (SX) message.
+ * These are text descriptions of problems affecting specific stops, routes, or other components
+ * of the transit system which will be displayed to users as text.
+ * Although they have flags describing the effect of the problem described in the text, convention
+ * is that these messages do not modify routing behavior on their own. They must be accompanied by
```

Comment:
I would mention that they _currently_ don't affect routing but we have talked about changing that several times, for example for stop closures.